From 61975464327f215f3259caba8531fc0f115b1f2d Mon Sep 17 00:00:00 2001
From: Mitch Luban <mluban@nvidia.com>
Date: Wed, 9 Jan 2013 16:21:28 -0800
Subject: [PATCH 02/10] power: Add POWER_HINT_APP_PROFILES to powerhal

This change adds POWER_HINT_APP_PROFILES to list of hints
available to powerhal.

Bug 1155948

Reviewed-on: http://git-master/r/190113
(cherry picked from commit af84456e8e31b5f47a72db08eb84113196977d89)

Change-Id: Ibe11110882490e19aa8eb9ad4258d29cf4b457ae
Reviewed-on: http://git-master/r/203503
Reviewed-by: Automatic_Commit_Validation_User
GVS: Gerrit_Virtual_Submit
Reviewed-by: Mitch Luban <mluban@nvidia.com>
Tested-by: Mitch Luban <mluban@nvidia.com>
Reviewed-by: Bharat Nihalani <bnihalani@nvidia.com>
---
 include/hardware/power.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index ee266f3..193ba68 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -41,6 +41,7 @@ __BEGIN_DECLS
 typedef enum {
     POWER_HINT_VSYNC = 0x00000001,
     POWER_HINT_INTERACTION = 0x00000002,
+    POWER_HINT_APP_PROFILE = 0x00000003,
     POWER_HINT_COUNT,
 } power_hint_t;
 
-- 
1.8.1.5

